srcopsmetrics/entities/readme.py

```diff
@@ -41,7 +43,9 @@ def analyse(self) -> List[GithubContentFile]:
         if self.previous_knowledge is None or len(self.previous_knowledge) == 0:
             return [self.get_raw_github_data()]
 
-        if self.previous_knowledge["readme"]["size"] == self.repository.get_readme().size:
+        readme = self.repository.get_readme()
+
+        if self.previous_knowledge[readme.path]["size"] == readme.size:
```

Comment:
what happens if readme does not exists here?

Reply:
So I've changed the readme to be stored based on it's `last_modified` timestamps, so that we can see the history of change in time. Also added the check if readme was deleted - we do not want to analyse it (yet?)

Reply:
why do we want to store the history of the readme? Maybe to create a new feature related to how oftern the docs are updated? I guess is an important factor for maintainability of the project. WDYT?

 I think can be a good dataset for a project in general. I was working on clustering data science packages from readme. I will put the dataset in `thoth-staton/datasets`

Reply:
I would say that the analogy is in the time - we store PRs and Issues, which means we store entities that are accumulating over the time. This is not case of a standalone readme in the root directory. So I thought maybe it would be better to store it over the time.

So you're suggesting a new entity e.g. `ReadMeHistory` that would do this?

Reply:
If we can gather some interesting features then I would say yes, otherwise for now we can leave as low priority.

Reply:
Aight, changed to not contain history :+1: